apps/frontline-widgets/src/app/messenger/components/conversation.tsx

```
@@ -0,0 +1,248 @@
+                />
+                <Avatar.Fallback>
+                  {supporter.details.firstName?.charAt(0) || 'C'}
+                </Avatar.Fallback>
+              </Avatar>
+            ))}
+        </AvatarGroup>
+        <span className="text-muted-foreground font-medium text-sm">
+          Our usual reply time
+        </span>{' '}
+        <mark className="bg-transparent text-primary font-medium text-sm">
+          ({`a few ${responseRate}` || 'a few minutes'})
```

ESLint

[error] Unexpected constant truthiness on the left-hand side of a `||` expression.

(no-constant-binary-expression)